pages/common/webpack.md

```diff
@@ -0,0 +1,19 @@
+# webpack
+
+> Bundle your js files and other assets into a single output file.
+
+- Create a single output file from an entry point file:
+
+`webpack {{app.js}} {{bundle.js}}`
+
+- Load css files too from your js file. (This uses the css loader for .css files):
+
+`webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
```

Comment:
Yea I know, it is kind of beyond the purview of a command line thing. Because you would be going into the contents of the js file. That's why, I did not know how to go about it. Where should I write that ?